ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -93,6 +97,50 @@ export default React.memo<Props>(function VideoJs(props: Props) {
 
   videoJsOptions.muted = startMuted;
 
+  const tapToUnmuteRef = useRef();
+
+  function showTapToUnmute(newState: boolean) {
+    // Use the DOM to control the state of the button to prevent re-renders.
+    // The button only needs to appear once per session.
+    if (tapToUnmuteRef.current) {
+      const curState = tapToUnmuteRef.current.style.visibility === 'visible';
+      if (newState !== curState) {
+        tapToUnmuteRef.current.style.visibility = newState ? 'visible' : 'hidden';
+      }
+    } else if (isDev) {
+      throw new Error('[videojs.jsx] Empty video ref should not happen');
```

Comment:
Instead of `electron-is-dev`, use `const isProduction = process.env.NODE_ENV === "production"`